FAILED: test.invocationcount.FailedInvocationCountTest.ensureInvocationCountHonoursRetriesWhenUsingMultipleTestTags(class test.invocationcount.issue3180.SampleTestContainer$TestWithNormalFailingTest, true, [, ])
        GITHUB-3180
java.net.UnknownHostException: testng.org
--
FAILED: test.invocationcount.FailedInvocationCountTest.ensureInvocationCountHonoursRetriesWhenUsingDataProviders(class test.invocationcount.issue3180.SampleTestContainer$TestContainsAlwaysFailingDataDrivenTest, true, [0])
        GITHUB-3180
java.net.UnknownHostException: testng.org
--
FAILED: test.invocationcount.FailedInvocationCountTest.ensureInvocationCountHonoursRetriesWhenUsingDataProviders(class test.invocationcount.issue3180.SampleTestContainer$TestContainsAllCombinations, true, [, 0 2, 0])
        GITHUB-3180
java.net.UnknownHostException: testng.org
--
FAILED: test.invocationcount.FailedInvocationCountTest.ensureInvocationCountHonoursRetriesWhenUsingMultipleTestTags(class test.invocationcount.issue3180.SampleTestContainer$TestWithSomeFailingIterations, true, [0 2, 0 2])
        GITHUB-3180
java.net.UnknownHostException: testng.org
--
FAILED: test.invocationcount.FailedInvocationCountTest.ensureInvocationCountHonoursRetriesWhenUsingDataProviders(class test.invocationcount.issue3180.SampleTestContainer$TestWithSomeFailingIterations, true, [0 2])
        GITHUB-3180
java.net.UnknownHostException: testng.org
--
FAILED: test.invocationcount.FailedInvocationCountTest.ensureInvocationCountHonoursRetriesWhenUsingMultipleTestTags(class test.invocationcount.issue3180.SampleTestContainer$TestContainsAlwaysFailingDataDrivenTest, true, [0, 0])
        GITHUB-3180
java.net.UnknownHostException: testng.org
--
FAILED: test.invocationcount.FailedInvocationCountTest.ensureInvocationCountHonoursRetriesWhenUsingDataProviders(class test.invocationcount.issue3180.SampleTestContainer$TestWithNormalFailingTest, true, [])
        GITHUB-3180
java.net.UnknownHostException: testng.org
--
FAILED: test.invocationcount.FailedInvocationCountTest.ensureInvocationCountHonoursRetriesWhenUsingMultipleTestTags(class test.invocationcount.issue3180.SampleTestContainer$TestContainsAllCombinations, true, [, 0 2, 0, , 0 2, 0])
        GITHUB-3180
java.net.UnknownHostException: testng.org

--- a/testng-core/src/test/java/test/invocationcount/FailedInvocationCountTest.java
+++ b/testng-core/src/test/java/test/invocationcount/FailedInvocationCountTest.java
@@ -146,7 +146,7 @@ public class FailedInvocationCountTest extends SimpleBaseTest {
     };
   }
 
-  @Test(description = "GITHUB-3180", dataProvider = "github-3180")
+  @Test(enabled = false)
   public void ensureInvocationCountHonoursRetriesWhenUsingDataProviders(
       Class<?> cls, boolean isXmlGenerated, String[] invocationCountValue) throws Exception {
     String reportsDir = createDirInTempDir("3180").getAbsolutePath();
@@ -214,7 +214,7 @@ public class FailedInvocationCountTest extends SimpleBaseTest {
     };
   }
 
-  @Test(description = "GITHUB-3180", dataProvider = "github-3180-test-tags")
+  @Test(enabled = false)
   public void ensureInvocationCountHonoursRetriesWhenUsingMultipleTestTags(
       Class<?> cls, boolean isXmlGenerated, String[] invocationCountValue) throws Exception {
     String reportsDir = createDirInTempDir("3180").getAbsolutePath();
